posts/start-web-performance-first-steps.md

```diff
@@ -0,0 +1,120 @@
+---
+title: 'Start tackling web performance issues in your company: the first steps'
+date: '2020-05-12'
+description: "The first steps we took to start tackling performance in QuintoAndar and how to speed them up"
+cover: 'https://lucianohgo.com/images/memoization.jpg'
+coverAlt: 'Memoization graph showing expensive vs cached renders'
+authors:
+  - lucianohgo
+tags:
+  - performance
+  - performance-culture
+  - product-management
+---
+
+When we're building our product, we're passionate about enabling users to do more, empowering them. This passion leads to the creation of more and more ways the users can use our products to better accomplish their goals. This rarely leaves us the time to tune and optimize and it's common to accumulate a lot of performance technical debt. **At some point, we realize as a company that [neglecting performance is directly affecting our business results](/posts/avoid-losing-users-and-conversions.md)**. When that day comes, what should we do?
+
+!["Road to a performant product a long asphalt road surrounded by trees"](/images/start-web-performance-first-steps/road.jpg "The road to a performant product is long, the sooner you start the better")
+*The road to a performant product is long, the sooner you start the better - Photo by [Matt Duncan](https://unsplash.com/@foxxmd?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText) on [Unsplash](https://unsplash.com/?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText)*
+
+
+First I'd like to start with a disclaimer: Not all of these were the first steps that we took, but rather they're assembled based on what we believe, given our current experience, would've been the best first steps to take, after all:
+
+> “Life can only be understood backwards, but it must be lived forward”
+>
+> -- [Søren Kierkegaard](https://en.wikipedia.org/wiki/S%C3%B8ren_Kierkegaard)
+
+The way we structured this post aims at better explaining why each of the steps is important and what it accomplishes:
+
+- [The Challenges](#the-challenges)
+- [The Steps](#the-steps)
+- [Step One: Measure and Lock](#step-one-measure-and-lock)
+  - [Measure: Real User Metrics and Conversion Metrics](#measure-real-user-metrics-and-conversion-metrics)
+  - [Lock: Audits and Continous Improvement](#lock-audits-and-continous-improvement)
+- [Step Two: Set up a Group](#step-two-set-up-a-group)
+- [Step Three: Make performance a part of the Company's Culture](#step-three-make-performance-a-part-of-the-companys-culture)
+- [In Summary](#in-summary)
+- [See Also](#see-also)
+
+Also, this post is focused on paying performance technical debt, but all these steps can be implemented from day one and actually help us shape a better product from the get-go.
+
+## The Challenges
+
+Already familiar with the challenges of tackling performance technical debt? [Go straight to the steps ⏭](#the-steps)
+
+
+## The Steps
```

Comment:
The topics you cover seems more like "topics" than "steps that occurs in a certain order", performance culture, for example, takes place all the way